### DISCUSSION TITLE
[V5] There is not websocket support?

### DISCUSSION THREAD

**Discussion by rodolphonetto** [adonisjs/core#1681] [Help] [answered]
Hey guys, I was looking for the websockets docs and did not found it out, there is no support yet or just the docs are not ready? if not, is it in the plans? or I need to use socket.io package for example?

Thanks

**Comment by yarapolana** [comment, marked as answer]
Hey @rodolphonetto 
According to Virk I don't think websockets will be migrated to v5 but you can use socket.io
Here is the official documentation on how to use it in v5.

https://preview.adonisjs.com/blog/socketio-with-adonisjs-v5

**Reply by rodolphonetto** [reply]
Thank you @yarapolana that is what I was looking for!
